src/hooks/engine.rs

```diff
@@ -204,6 +210,10 @@ pub static LoadEntityDLLs: Pointer<unsafe extern "C" fn(*const c_char)> = Pointe
     ]),
     my_LoadEntityDLLs as _,
 );
+
+pub static Mod_LeafPVS: Pointer<unsafe extern "C" fn(*mut mleaf_s, *mut model_s) -> *mut c_void> =
+    Pointer::empty(b"Mod_LeafPVS\0");
+
```

Comment:
```suggestion
pub static Mod_LeafPVS: Pointer<unsafe extern "C" fn(*mut mleaf_s, *mut model_s) -> *mut c_void> =
    Pointer::empty(b"Mod_LeafPVS\0");
```
I also removed the blank lines in my suggestion (although github seems to eat the last one for some reason)

Reply:
github'd